doc/source/examples/kernel_shap_adult_lr.ipynb

```diff
@@ -644,11 +644,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from alibi.explainers.shap_wrappers import sum_categories"
+    "from alibi.explainers import sum_categories"
```

Comment:
We'll need to think more where to put these essentially utility functions. Don't want to pollute the `explainers` namespace too much, so previously importing directly from `shap_wrappers` worked quite well.

Reply:
Ok, I've reverted this so that `sum_categories` is imported from `shap_wrappers` again. I did the same with `plot_ale` and `Callback`.

Reply:
I think we should keep the `plot_ale` and `Callback` where they are for now, otherwise we risk breaking stuff? I don't think there's a clean way of exporting them in `alibi.explainers` but raising a warning if it's imported from there rather than the specific modules.

Reply:
Ah, yeah that makes sense. `Callback` was originally imported from the `counterfactual.cfrl_base`. I'll move the `plot_ale` back though!